sqlvm-alwayson-cluster/nested/provisioningVM1.json

```diff
@@ -0,0 +1,412 @@
+{  
```

Comment:
This is a very useful info, I have created a VSTS item for this improvement and i will follow up on it using this dynamical data disk selector. For this initial check-in i will stay with the current solution since it has been tested out for many times hence a higher reliability.